tests/frameworks/include/Helpers/TestHelpers.h

```diff
@@ -29,6 +29,7 @@
 @interface THBooleanCondition : NSObject
 - (BOOL)waitUntilDate:(NSDate*)limit;
 - (void)broadcast;
+- (void)signal;
```

Comment:
This conforms to the original implementation of NSURLSession tests... and it more completely fills out `THBooleanCondition`'s implementation of `NSCondition`